tests/libres_tests/res/enkf/test_local_config.py

```diff
@@ -75,7 +75,7 @@ def test_local_obs_data(self):
             self.assertTrue(node_added)
 
             # Error when adding existing obs node
-            with self.assertRaises(KeyError):
+            with self.assertRaises(Exception):
```

Comment:
You can use `throw py::key_error` in pybind11 C++ code to get a `KeyError` here

Reply:
That I did not know - that was very valuable information.